### DISCUSSION TITLE
Drawing Chinese text using DrawText on Blazor WebAssembly is not sharp

### DISCUSSION THREAD

**Discussion by qwertzou** [mono/SkiaSharp#2708] [Q&A]
Compared to WinForm, the effect is significantly different
![3554249050](https://github.com/mono/SkiaSharp/assets/33852145/6f86653a-dc73-48fa-ac07-3153674a6bab)
![3880687909](https://github.com/mono/SkiaSharp/assets/33852145/4b0805a7-8398-4666-9e43-1b18d1e0a8de)

**Comment by qwertzou** [comment]
The font used is SimSun

**Reply by taublast** [reply]
While this was about windows could you please check if this helps? https://github.com/mono/SkiaSharp/issues/830#issuecomment-841225721

**Reply by qwertzou** [reply]
Thank you for your answer. I tried, but it didn't help me
